### ISSUE TITLE
description isn't defined when used as parameter

### ISSUE THREAD

**Issue by fabriziofortino** [foundweekends/giter8#321]
I have created a PR(#320) with the steps to reproduce the issue.

It seems that when `description` is used as parameter it does not get correctly recognized / applied in the templates. As a workaround I have renamed the property to `desc`. Is `description` a kind of reserved key?

```
File: /var/folders/22/g7d85p4j2cx7bj3sf2mb0tg489wxn0/T/g8test-1552970002210451/template/src/main/g8/foo.txt, context [anonymous] 1:7 internal error: org.stringtemplate.v4.compiler.STException: context [anonymous] 1:7 attribute description isn't defined
	at giter8.G8$STErrorHandler.runTimeError(g8.scala:117)
	at giter8.G8$STErrorHandler.runTimeError(g8.scala:112)
	at org.stringtemplate.v4.misc.ErrorManager.runTimeError(ErrorManager.java:133)
	at org.stringtemplate.v4.Interpreter._exec(Interpreter.java:193)
	at org.stringtemplate.v4.Interpreter.exec(Interpreter.java:145)
	at org.stringtemplate.v4.ST.write(ST.java:427)
	at org.stringtemplate.v4.ST.render(ST.java:497)
	at org.clapper.scalasti.ST.render(ST.scala:285)
	at giter8.G8$.applyTemplate(g8.scala:109)
	at giter8.G8$.write(g8.scala:160)
	at giter8.G8$.write(g8.scala:145)
	at giter8.G8$$anonfun$writeTemplates$2$$anonfun$apply$11.apply(g8.scala:424)
	at giter8.G8$$anonfun$writeTemplates$2$$anonfun$apply$11.apply(g8.scala:424)
	at scala.util.control.Exception$Catch$$anonfun$opt$1.apply(Exception.scala:118)
	at scala.util.control.Exception$Catch$$anonfun$opt$1.apply(Exception.scala:118)
	at scala.util.control.Exception$Catch.apply(Exception.scala:102)
	at scala.util.control.Exception$Catch.opt(Exception.scala:118)
	at giter8.G8$$anonfun$writeTemplates$2.apply(g8.scala:423)
	at giter8.G8$$anonfun$writeTemplates$2.apply(g8.scala:413)
	at scala.collection.immutable.Stream.foreach(Stream.scala:547)
	at giter8.G8$.writeTemplates(g8.scala:413)
	at giter8.G8$$anonfun$applyT$1.apply(g8.scala:272)
	at giter8.G8$$anonfun$applyT$1.apply(g8.scala:265)
	at scala.util.Either$RightProjection.flatMap(Either.scala:523)
	at giter8.G8$.applyT(g8.scala:265)
	at giter8.G8$.fromDirectory(g8.scala:62)
	at giter8.IntegrationTestHelpers$$anonfun$checkGeneratedProject$1.apply$mcV$sp(IntegrationTestHelpers.scala:12)
	at giter8.IntegrationTestHelpers$$anonfun$checkGeneratedProject$1.apply(IntegrationTestHelpers.scala:12)
	at giter8.IntegrationTestHelpers$$anonfun$checkGeneratedProject$1.apply(IntegrationTestHelpers.scala:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:57)
	at scala.Console$.withIn(Console.scala:216)
	at giter8.IntegrationTestHelpers$class.checkGeneratedProject(IntegrationTestHelpers.scala:11)
	at giter8.IntegrationTest.checkGeneratedProject(IntegrationTest.scala:8)
	at giter8.IntegrationTest$$anonfun$12$$anonfun$apply$mcV$sp$12.apply(IntegrationTest.scala:104)
	at giter8.IntegrationTest$$anonfun$12$$anonfun$apply$mcV$sp$12.apply(IntegrationTest.scala:99)
	at giter8.IntegrationTest$$anonfun$giter8$IntegrationTest$$testCase$1.apply(IntegrationTest.scala:150)
	at giter8.IntegrationTest$$anonfun$giter8$IntegrationTest$$testCase$1.apply(IntegrationTest.scala:146)
	at giter8.TestFileHelpers$.tempDirectory(TestFileHelpers.scala:11)
	at giter8.IntegrationTest.giter8$IntegrationTest$$testCase(IntegrationTest.scala:146)
	at giter8.IntegrationTest$$anonfun$12.apply$mcV$sp(IntegrationTest.scala:99)
	at giter8.IntegrationTest$$anonfun$12.apply(IntegrationTest.scala:99)
	at giter8.IntegrationTest$$anonfun$12.apply(IntegrationTest.scala:99)
	at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
	at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
	at org.scalatest.Transformer.apply(Transformer.scala:22)
	at org.scalatest.Transformer.apply(Transformer.scala:20)
	at org.scalatest.FlatSpecLike$$anon$1.apply(FlatSpecLike.scala:1682)
	at org.scalatest.TestSuite$class.withFixture(TestSuite.scala:196)
	at org.scalatest.FlatSpec.withFixture(FlatSpec.scala:1685)
	at org.scalatest.FlatSpecLike$class.invokeWithFixture$1(FlatSpecLike.scala:1679)
	at org.scalatest.FlatSpecLike$$anonfun$runTest$1.apply(FlatSpecLike.scala:1692)
	at org.scalatest.FlatSpecLike$$anonfun$runTest$1.apply(FlatSpecLike.scala:1692)
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:289)
	at org.scalatest.FlatSpecLike$class.runTest(FlatSpecLike.scala:1692)
	at org.scalatest.FlatSpec.runTest(FlatSpec.scala:1685)
	at org.scalatest.FlatSpecLike$$anonfun$runTests$1.apply(FlatSpecLike.scala:1750)
	at org.scalatest.FlatSpecLike$$anonfun$runTests$1.apply(FlatSpecLike.scala:1750)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:396)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:373)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:410)
	at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:379)
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:461)
	at org.scalatest.FlatSpecLike$class.runTests(FlatSpecLike.scala:1750)
	at org.scalatest.FlatSpec.runTests(FlatSpec.scala:1685)
	at org.scalatest.Suite$class.run(Suite.scala:1147)
	at org.scalatest.FlatSpec.org$scalatest$FlatSpecLike$$super$run(FlatSpec.scala:1685)
	at org.scalatest.FlatSpecLike$$anonfun$run$1.apply(FlatSpecLike.scala:1795)
	at org.scalatest.FlatSpecLike$$anonfun$run$1.apply(FlatSpecLike.scala:1795)
	at org.scalatest.SuperEngine.runImpl(Engine.scala:521)
	at org.scalatest.FlatSpecLike$class.run(FlatSpecLike.scala:1795)
	at org.scalatest.FlatSpec.run(FlatSpec.scala:1685)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$1.apply(Runner.scala:1340)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$1.apply(Runner.scala:1334)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1334)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:1011)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:1010)
	at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1500)
	at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1010)
	at org.scalatest.tools.Runner$.run(Runner.scala:850)
	at org.scalatest.tools.Runner.run(Runner.scala)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.runScalaTest2(ScalaTestRunner.java:138)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.main(ScalaTestRunner.java:28)
```

**Comment by xuwei-k** [comment]
https://github.com/foundweekends/giter8/blob/v0.9.0/library/src/main/scala/g8.scala#L353

**Comment by TonioGela** [comment]
Fixed in #572. Can be closed.
